src/clients/node/package.json

```diff
@@ -13,19 +13,10 @@
     "LICENSE",
     "README.md",
     "dist",
-    "dist/.client.node.sha256",
```

Comment:
Should bump package version

Reply:
Eventually we also need a check that makes sure that the versions in package.json and package-lock.json are the same.

Could do with:

```
npm i
./scripts/fail_on_diff.sh
```